back/tests/WebSpec.hs

```diff
@@ -48,9 +46,6 @@ tests = withSystemTempFile "test_guide.log" $ \logFile logFileHandle -> do
     itemTests
     markdownTests
     ApiSpec.tests
-  -- TODO: get rid of the 'threadDelay' by using 'withAsync' instead of
-  -- 'Slave.fork'
-  threadDelay 1000000
```

Comment:
Does this actually work? Because in other places we still have `Slave.fork` and I'm not sure if it mixes with `withAsync`. If it does, great.

Reply:
Tests passed. A site server works. Is it actually? Likely, it is.
What about to swap `Slave.fork` with `withAsync`